src/cap-description.adoc

```diff
@@ -135,9 +135,9 @@ Therefore, it is only possible to encode a subset of all combinations.
 .Permissions widths depending on MXLEN
 [#perms_bit_width,options=header,align="center",width="80%",cols="^13%,^27%,<60%"]
 |==============================================================================
-^| MXLEN | Permissions width      | Comment
-^| 32    | {cap_rv32_perms_width} | Encodes some combinations of 5 permission bits and _includes_ the <<m_bit>> in quadrant 1 (see <<cap_perms_encoding32>>)
-^| 64    | {cap_rv32_perms_width} | Separate bits for each permission, _excluding_ the <<m_bit>>
+^| MXLEN | AP field width         | Comment
+^| 32    | {cap_rv32_perms_width} | Encodes some combinations of 5 permission bits, including the <<m_bit>> if {cheri_default_ext_name} is supported.
```

Comment:
There is a hard-coded 5 here. I think its best to remove it or make it {cap_rv64_perms_width}